src/functions/template-tags/venue.php

```diff
@@ -422,6 +422,22 @@ function tribe_get_venue_website_link( $post_id = null, $label = null ) {
 	}
 
 	/**
+	 * Returns the venue website URL related to the current post or for the optionally
+	 * specified post.
+	 *
+	 * @param int|null $post_id
+	 *
+	 * @return string
+	 */
+	function tribe_get_venue_website_url( $post_id = null ) {
+		return (string) tribe_get_event_meta(
```

Comment:
Can we make a note to talk about how to split function calls like this one in a Code Review meeting? It can get really weird, I mean we should decide on something, because some of us are doing X and others are doing Y, and on different cases.